### ISSUE TITLE
'pip-purge' gives error on macos

### ISSUE THREAD

**Issue by peeush-the-developer** [ParthS007/pip-purge#9]
Hello,
I installed `pip-purge` using pip, then used `pip-purge` but it gives me error, as following:
```
~ % pip3 install pip-purge
Collecting pip-purge
  Downloading pip_purge-0.1.2-py3-none-any.whl (2.6 kB)
Collecting envoy
  Downloading envoy-0.0.3-py2.py3-none-any.whl (5.0 kB)
Installing collected packages: envoy, pip-purge
Successfully installed envoy-0.0.3 pip-purge-0.1.2
~ % pip-purge
  File "/opt/homebrew/bin/pip-purge", line 15
    print(f"Found {installed} dirty packages installed.")
TabError: inconsistent use of tabs and spaces in indentation
```

Could you please help me in resolving this?

Thanks in advance

**Comment by reddytocode** [comment]
If you are in a virtual env you can run 
```virtualenv --clear venv```
or instead of venv you virtualenv name

**Comment by gawbul** [comment]
There is an issue with the line indentations in the package. It needs fixing and pushing to pypi again.

**Comment by gawbul** [comment]
> Hello,
> I installed `pip-purge` using pip, then used `pip-purge` but it gives me error, as following:
> 
> ```
> ~ % pip3 install pip-purge
> Collecting pip-purge
>   Downloading pip_purge-0.1.2-py3-none-any.whl (2.6 kB)
> Collecting envoy
>   Downloading envoy-0.0.3-py2.py3-none-any.whl (5.0 kB)
> Installing collected packages: envoy, pip-purge
> Successfully installed envoy-0.0.3 pip-purge-0.1.2
> ~ % pip-purge
>   File "/opt/homebrew/bin/pip-purge", line 15
>     print(f"Found {installed} dirty packages installed.")
> TabError: inconsistent use of tabs and spaces in indentation
> ```
> 
> Could you please help me in resolving this?
> 
> Thanks in advance

Another way to do this without this package is something similar to the following:

```
pip freeze | cut -d = -f 1 | xargs sudo pip3 uninstall -y
```

**Comment by ParthS007** [comment]
fixed with with #10 . Thanks for reporting.
